README.md

```diff
@@ -17,63 +17,78 @@ Answer the following data queries. Keep track of the SQL you write by pasting it
-> This can be done with SELECT, COUNT, JOIN and GROUP BY clauses. Your count should focus on a field in the Orders table, not the Customer table
+> SELECT count(o.customerID), c.customerName
+> FROM orders o JOIN customers c
+> WHERE o.customerID = c.customerid
+> GROUP BY o.customerID
 
-> There is more information about the COUNT clause on [W3 Schools](https://www.w3schools.com/sql/sql_count_avg_sum.asp)
 
 ### list customers names and the number of orders per customer. Sort the list by number of orders in descending order. _Ernst Handel_ should be at the top with 10 orders followed by _QUICK-Stop_, _Rattlesnake Canyon Grocery_ and _Wartian Herkku_ with 7 orders each.
-> This can be done by adding an ORDER BY clause to the previous answer
+> SELECT count(o.customerID), c.customerName
+> FROM orders o JOIN customers c
+> WHERE o.customerID = c.customerid
+> GROUP BY o.customerID
+> ORDER BY count(o.customerID) DESC
 
 ### list orders grouped by customer's city showing number of orders per city. Returns 58 Records with _Aachen_ showing 2 orders and _Albuquerque_ showing 7 orders.
-> This is very similar to the previous two queries, however, it focuses on the City rather than the CustomerName
+> SELECT count(o.customerID), c.city
+> FROM orders o JOIN customers c
+> WHERE o.customerID = c.customerid
+> GROUP BY c.city
+> ORDER BY count(o.customerID) DESC
 
 ## Stretch Goals
 
 ### delete all customers that have no orders. Should delete 17 (or 18 if you haven't deleted the record added) records.
-> This is done with a DELETE query
-
-> In the WHERE clause, you can provide another list with an IN keyword this list can be the result of another SELECT query. Write a query to return a list of CustomerIDs that meet the criteria above. Pass that to the IN keyword of the WHERE clause as the list of IDs to be deleted
- 
-> Use a LEFT JOIN to join the Orders table onto the Customers table and check for a NULL value in the OrderID column
+> DELETE
+> FROM customers
+> WHERE customerid IN
+> (SELECT c.customerid
+> FROM customers c LEFT JOIN orders o
+> ON o.CustomerID = c.CustomerID  
+> WHERE o.orderid IS NULL)
 
 ## Create Database and Table
 
 ### Keep track of the code you write and paste at the end of this document
 
-- use pgAdmin to create a database, naming it `budget`.
-- add an `accounts` table with the following _schema_:
-
-  - `id`, numeric value with no decimal places that should autoincrement.
-  - `name`, string, add whatever is necessary to make searching by name faster.
-  - `budget` numeric value.
-
-- constraints
-  - the `id` should be the primary key for the table.
-  - account `name` should be unique.
-  - account `budget` is required.
+> CREATE TABLE public.accounts
+> (
+>     id integer NOT NULL GENERATED BY DEFAULT AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
+>     name text COLLATE pg_catalog."default" NOT NULL,
+>     budget integer,
+>     CONSTRAINT "unique" UNIQUE (name, id)
+
+> )
```

Comment:
Line 84 - you might want to use a data type that supports decimal values (perhaps something like `DOUBLE` or `DECIMAL`).